fastlane_bot/events/exceptions.py

```diff
@@ -4,4 +4,11 @@ def __init__(self, filepath):
 
     def __str__(self):
         return (f"tokens.csv does not exist at {self.filepath}. Please run the bot without the `read_only` flag to "
-                f"create this file.")
\ No newline at end of file
+                f"create this file.")
+
+
+class AyncUpdateRetryException(Exception):
```

Comment:
`AyncUpdateRetryException` seems to be missing an "s"

Reply:
custom implementation - it's a child of Exception, no need for an s imo